Resources/Prototypes/Entities/Clothing/OuterClothing/misc.yml

```diff
@@ -377,3 +377,14 @@
     sprite: Clothing/OuterClothing/Misc/red_racoon.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Misc/red_racoon.rsi
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterUnathiRobe
+  name: roughspun robes
+  description: A traditional Unathi garment.
```

Comment:
We don't have Unathi on EE (yet?), we have Lizardpeople.

Reply:
They are the same.

Reply:
I am aware, change the name of the clothing or change the name of the species.

Reply:
> I am aware, change the name of the clothing or change the name of the species.